src/screens/welcome/Welcome.js

```diff
@@ -94,19 +113,28 @@ class Welcome extends Component {
     this.props.resetStory();
     this.props.resetMap();
     this.props.resetPhoto();
+    // Fetch the list of file paths
     await this.stateManager.getFilePaths();
+    // If the list of file paths is empty, set this.unfinishedReportsAbsent false
```

Comment:
this is a bit too literal of a comment - i'd suggest deleting this comment and changing the comment above to "fetches list of files paths of all in-progress reports on device, if any"